### ISSUE TITLE
wk3 - Flesh out systems thinking stories by level for all 5 modules

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#1568]
@branscombj will use the file **systems_thinking_codebook_2018_10_16.xlsx** from the OSF project (https://osf.io/sf574/) to flesh out the systems thinking stories across each of the 4 levels of CFBT (Complexity, Feedback, Behavior, Time) Systems Thinking in the OSF project here: r21_qual_systems_thinking - https://osf.io/36syr/

**Comment by branscombj** [comment]
Ack. @staceypark @lijenn - I was derailed for a couple of weeks and now, once again when I come try to work on this task, the assignment eludes me. In the top comment, 3 spreadsheets are referenced. They're all called Systems Thinking Codebook. Can you please remind me what exactly I'm supposed to do and I really will work on it this week! 😣 Sorry to be so dense 🧱

**Comment by branscombj** [comment]
@dlounsbu Could you and I meet and do some work on this together? I'm puzzled by some of the existing stories and not clear on Level 3 vs 4, especially on stories that aren't actually feedback loops in our models (like Overbooks).

**Comment by dlounsbu** [comment]
@branscombj : Yes, we can connect on this. I could check in with you via teams between 2pm and 2:30pm.

**Comment by staceypark** [comment]
@branscombj and @dlounsbu

Thanks for meeting about this today!
Jane let me know that you guys made a game plan for the next 2 weeks to get this completed.🚀🎆 
Could one person document that plan here on GitHub as well? 

fyi: @lzim

**Comment by branscombj** [comment]
@dlounsbu and I discussed the short definitions of Levels 1-4 ST in the spreadsheet, which are hard to clearly differentiate. 

- [ ] I am going to add the CFBT coding details to those headers for additional guidance. 
- [ ] David will write Level 1-4 example stories for the Higher Care Quality Improves Recovery loop in SP.
- [ ] I will write Level 1-4 stories for Overwork and Burnout Reduce Quality (based on @TomRust 's input (see #1511).
- [ ] I will review the rest of the spreadsheet and see if I can fill in any gaps or improve any of the examples so they better demonstrate the progression through the Levels.

We will check out/in the Codebook https://osf.io/k625q/ to do this work.

I am pondering whether there may be some team questions that are not related to feedback loops, e.g. Overbooking Affects No-Shows, such that teams prioritizing these won't be able to develop or demonstrate Levels 2-4 systems thinking - all of which require closing a loop.

fyi @staceypark @lzim

**Comment by branscombj** [comment]
@lzim @dlounsbu Do we anticipate any teams wanting to experiment in parts of the system that do not involve feedback loops, e.g. Overbooking Affects No-Shows? Our definitions for ST levels depend on closing loops, so if that's the case, those teams will not be able to demonstrate Levels 2-3-4 ST.

**Comment by branscombj** [comment]
Hi @dlounsbu ! I have done 2 of the items above and created a row for your 4 ST examples for Higher Care Quality Improves Recovery loop in SP. Please see what you think of what I put. I used red font to highlight words that will need to change because they don't currently match what's in the sim. (See my comments in #1511 about that.)
Cheers!

**Comment by lzim** [comment]
Cross-ref: #1510.

**Comment by staceypark** [comment]
#3 Priority #1568 - This best builds on work for #1510 & #1511.
We need to finishing the Systems Thinking codebook for each module.

**Comment by lzim** [comment]
@staceypark 
@branscombj 

Are #1510 & #1511 ready to close?
Where does this fall next?

I'm asking because I'm wondering if we can include these in the _MTL Facilitator Manual_

Thanks!

**Comment by staceypark** [comment]
from 2/22 mm meeting:
after wrapping #1510, @branscombj is going to work on this with the goal to hand over to @lzim sometime Friday to add to MTL Facilitator manual.

**Comment by lzim** [comment]
@branscombj 

Did you have an update on these for inclusion in the _MTL_ Bookdown manual? 
I’m going to be working on it over the weekend.


Thanks!

**Comment by branscombj** [comment]
> @branscombj
> 
> Did you have an update on these for inclusion in the _MTL_ Bookdown manual?
> I’m going to be working on it over the weekend.
> 
> Thanks!

Hi @lz I am working on it this weekend also. I had several questions about the AGG map in #1510 that I've never gotten responses on. I transferred all of my feedback loop stories into a Word table to make the ones that I think are OK/final as well as the questions more visible than scattered through the GH thread. I'll move on to the ST codebook CFBT examples now!

**Comment by branscombj** [comment]
> > @branscombj
> > Did you have an update on these for inclusion in the _MTL_ Bookdown manual?
> > I’m going to be working on it over the weekend.
> > Thanks!
> 
I uploaded a new spreadsheet with just CFBT Examples for the 5 models, leaving all of the other systems stories in the qual codebook as they were.

**Comment by hnorthcraft** [comment]
#adjudicate_memo adding research VS
